doc/user-guide/io.rst

```diff
@@ -898,6 +898,7 @@ where the data should be written (in index space, not label space), e.g.,
 
 Concurrent writes with ``region`` are safe as long as they modify distinct
 chunks in the underlying Zarr arrays (or use an appropriate ``lock``).
+Note that incorrectly (or omitting) setting the ``chunksizes`` for the initial Zarr file results in a partial update of the chunks when writing to regions, causing missing values in the Zarr file.
```

Comment:
I don't know enough about writing to regions to comment on the content, so here are some style suggestions:
```suggestion
Note that incorrectly setting (or omitting) the ``chunksizes`` for the initial
Zarr file results in a partial update of the chunks when writing to regions,
causing missing values in the Zarr file.
```
We could also use `.. note::` for this if we think this is important enough.